setup.py

```diff
@@ -66,8 +66,15 @@
                                   'filter_fdb_input/*',
                                   'pfcwd_input/*',
                                   'sku_create_input/*',
-                                  'sku_create_input/ACS-MSN2700/*',
-                                  'sku_create_input/Mellanox-SN2700-D48C8/*']
+                                  'sku_create_input/2700_files/*',
+                                  'sku_create_input/3800_files/*',
+                                  'sku_create_input/port_split_files/*',
+                                  'sku_create_input/port_unsplit_files/*',
+                                  'sku_create_input/2700_files/ACS-MSN2700/*',
+                                  'sku_create_input/2700_files/Mellanox-SN2700-D48C8/*',
+                                  'sku_create_input/2700_files/Mellanox-SN2700-C28D8-ORIG/*',
+                                  'sku_create_input/3800_files/ACS-MSN3800/*',
+                                  'sku_create_input/3800_files/Mellanox-SN3800-D28C50/*']
```

Comment:
This is OK for now, but hard to maintain. For the future we should look into a way of adding directories recursively, which is apparently not easy in setup.py. :(

Some discussion here for posterity: https://stackoverflow.com/questions/40031422/python-recursively-include-package-data-in-setup-py